app/views/finance/provider/settings/show.html.slim

```diff
@@ -1,135 +1,113 @@
-              include_blank: false
-
-    = f.actions do
-      = f.commit_button 'Save'
-
-- if can?(:manage, :finance)
-  - if @account.billing_strategy.try(:needs_credit_card?)
-    = semantic_form_for @account, url: admin_account_payment_gateway_path,
-                        html: { id: 'payment-gateway-form' } do |form|
-      h2 Credit card gateway
-      p Select your credit card gateway and configure it with the details of your merchant account.
-
-      = form.inputs "Basics" do
-        li class="select"
-          = form.label :payment_gateway_type, 'Gateway'
-          = form.collection_select :payment_gateway_type, @payment_gateways,
-                                   :type, :display_name, include_blank: "Choose one"
-          - if @current_gateway&.deprecated?
-            p class="InfoBox InfoBox--notice"
-              ' The
-              => @account.payment_gateway_type.capitalize
-              | gateway has been deprecated. It can still be used but you won't be able to switch back to it once you confirmed the switch to another gateway.
-
-      = form.fields_for :payment_gateway_options do |fields|
-        - @payment_gateways.each do |payment_gateway|
-          - enabled = payment_gateway.type == @account.payment_gateway_type
-          div class="payment_gateway_options" id="payment_gateway_#{payment_gateway.type}" style="display: #{enabled ? 'block' : 'none'}"
-
-            - if payment_gateway.fields.present?
-              = form.inputs "#{link_to payment_gateway.display_name, payment_gateway.homepage_url} Options" do
-                - payment_gateway.fields.each do |name, label|
-                  li class="string"
+- if @current_gateway&.deprecated?
+  - content_for :page_header_alert do
+    br
+    = pf_inline_alert t('.deprecation_warning', gateway: @current_gateway.type.capitalize), variant: :warning
+
+div class="pf-l-flex pf-m-column"
+  div class="pf-l-flex__item"
+    div class="pf-c-card"
+      div class="pf-c-card__title" Billing mode
+      div class="pf-c-card__body"
+        = semantic_form_for @billing_strategy, url: admin_finance_billing_strategy_path,
+                                               method: :put,
+                                               html: { class: 'pf-c-form pf-m-limit-width' } do |form|
+          div class="pf-c-content"
+            = t(".#{@billing_strategy.name}_description_html")
+
+          - new_mode = @billing_strategy.is_a?(Finance::PostpaidBillingStrategy) ? 'prepaid' : 'postpaid'
+          = hidden_field_tag 'type', new_mode
+          div class="pf-c-form__group pf-m-action"
+            div class="pf-c-form__actions"
+              button class="pf-c-button pf-m-danger" type="submit" data={ confirm: t('.switch_confirm', mode: new_mode) }
+                = t('.switch_button', mode: new_mode.upcase)
+
+  div class="pf-l-flex__item"
+    div class="pf-c-card"
+      div class="pf-c-card__title" Billing Settings
```

Comment:
Just a nitpick :grimacing: 

"Billing Settings" uses title case, while "Billing mode" and "Credit card gateway" use sentence case.
I'm not quite sure what we use in other pages, probably a mix also, but within the same page I think we need to use the same style :grimacing: I personally think sentence case looks better here.

Reply:
Apart from this, looks great!